pkg/expr/classic/classic.go

```diff
@@ -11,52 +11,44 @@ import (
 	"github.com/grafana/grafana/pkg/expr/mathexp"
 )
 
-// ConditionsCmd is command for the classic conditions
-// expression operation.
-type ConditionsCmd struct {
+// ClassicConditionsCmd is a command that supports the reduction and comparison of conditions.
+//
+// A condition in ClassicConditionsCmd can reduce a time series; or contain either an instant metric
```

Comment:
I think this paragraph/sentence is too complicated. Can you split the ';' apart. A bulleted list might be nice. The three things it can be/do aren't all that clear.

A stab at it (I think it's OK to briefly explain what a `condition` is here:
```
Each condition in the command contains a query reference, a reducer, and an evaluator. The QueryReference can refer to a time series, an instant metric, or the result of another expression. The condition checks whether evaluator(reducer(query reference)) is true or false.
```